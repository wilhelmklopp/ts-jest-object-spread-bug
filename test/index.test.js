const index = require('../lib');

test('index works', () => {
  expect(index()).toMatchSnapshot()
})
