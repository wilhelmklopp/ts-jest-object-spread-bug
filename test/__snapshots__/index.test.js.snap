// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index works 1`] = `
Object {
  "bug": "can't",
  "come": "to the phone right now",
  "why?": "Oh, cause she's dead!",
}
`;
